pennylane/transforms/control.py

```diff
@@ -129,11 +129,6 @@ def adjoint(self):
                 # Execute all ops adjointed.
                 ops = adjoint(requeue_ops_in_tape)(self.subtape)
 
-            if not new_tape.operations:
-                with qml.tape.QuantumTape() as new_tape:
-                    for op in ops:
-                        op.queue()
-
         return ControlledOperation(new_tape, self.control_wires)

```

Comment:
This has been removed since by setting the default adjoint to `True`, this code will never be visited.